app/src/main/java/com/eveningoutpost/dexdrip/StartNewSensor.java

```diff
@@ -160,21 +160,45 @@ public void onTimeUpdated(int newmins) {
                     ucalendar.add(Calendar.HOUR_OF_DAY, -1); // hack for warmup time
                 }
 
-                realStartSensor();
+                startSensorOrAskForG6Code();
             }
         });
         timePickerFragment.show(activity.getFragmentManager(), "TimePicker");
     }
 
-    private void realStartSensor() {
+    private void startSensorOrAskForG6Code() {
         if (Ob1G5CollectionService.usingCollector() && Ob1G5StateMachine.usingG6()) {
-            G6CalibrationCodeDialog.ask(this, this::realRealStartSensor);
+            G6CalibrationCodeDialog.ask(this, this::startSensorAndSetIntent);
         } else {
-            realRealStartSensor();
+            startSensorAndSetIntent();
         }
     }
 
+    private void startSensorAndSetIntent() {
```

Comment:
Is this just a rename or are there deeper changes going on here?

Reply:
The cleanup in this file was just a rename from:
* realStartSensor => startSensorOrAskForG6Code
* realRealStartSensor => startSensorAndSetIntent

The former realStartSensor only asks for a G6 code if necessary and then calls realRealStartSensor, and the ambiguity in the names was bothering me a bit